recipe/meta.yaml

```diff
@@ -8,10 +8,12 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://github.com/linbox-team/linbox/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 47f025fe7aab204fb276307f32a562de357c887c6ef589b9e4f7a57a3b58ecaf
+  sha256: 7b69ab617b15ca6684b29d08531ed8899cd4d07ec75799e5d624c4c0a60dc6b2
+  patches:
+    - patches/9e6e5dd-f4b483c.patch
```

Comment:
Can you add a comment here about where this patch is coming from?

Reply:
Something like this?